src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function TodoForm(props) {
+  return (
+    <form className="form" onSubmit={e => props.addTask(e)}>
```

Comment:
```suggestion
    <form className="form" onSubmit={props.addTask}>
```
This is fine, but I just wanted to show you that you don't need to write an anonymous function here